package.json

```
@@ -86,7 +86,7 @@
     "bench:server": "node benchmarks/server.js",
     "prebench:run": "node benchmarks/wait.js",
     "bench:run": "SAMPLES=100 CONNECTIONS=50 node benchmarks/benchmark.js",
-    "serve:website": "docsify serve .",
+    "serve:website": "echo 'Documentation has been moved to `\/documentation`'",
```

maybe more like the test script when you initialize a project with npm init?

```suggestion
    "serve:website": "echo \"Error: Documentation has been moved to '\/documentation'\" && exit 1",
```